### PR TITLE
fix(overflow-menu): stop updating items outside of the angular change detection

### PR DESCRIPTION
**Pull request by cal-smith**

### CODE REVIEW

File: src/dialog/overflow-menu/overflow-menu-option.component.ts [hunk 1]
```diff
@@ -61,29 +62,18 @@ export class OverflowMenuOption {
 
 	public tabIndex = -1;
 
+	public title = null;
```
Comment (zvonimirfras) [inline src/dialog/overflow-menu/overflow-menu-option.component.ts]: Maybe this can be a getter so it updates automatically like it used to?

Comment (cal-smith) [reply src/dialog/overflow-menu/overflow-menu-option.component.ts]: Nope, that was also causing some change after check errors ... the component instance shouldn't be reused in this case anyway, and if it is I'd expect `AfterViewInit` to be called again

Comment (zvonimirfras) [reply src/dialog/overflow-menu/overflow-menu-option.component.ts]: Cool. Do you mind adding that explanation as a comment?

Comment (cal-smith) [reply src/dialog/overflow-menu/overflow-menu-option.component.ts]: Yep 👍

**Pull request merged**